packages/pyinstaller/bitmessagemain.spec

```diff
@@ -72,8 +87,18 @@ exe = EXE(pyz,
           a.zipfiles,
           a.datas,
           a.binaries,
+          [],
           name=fname,
-          debug=False,
+          debug=all,
           strip=None,
-          upx=False,
-          console=False, icon= os.path.join(srcPath, 'images', 'can-icon.ico'))
+          upx=True,
```

Comment:
I remember people complained that this makes it slow. What is the size of the .exe with `upx=False,`?